config/default.schema.json

```diff
@@ -453,6 +448,11 @@
 			"minimum": 0,
 			"default": 8000,
 			"description": "Default timeout for external requests in milliseconds."
+		},
+		"LOG_LEVEL": {
+			"type": "string",
+			"default": "debug",
```

Comment:
add enum:[debug, emerg, error] add levels (numbers) from globals.js and cleanup that file Lines 12...24